vendor/sigs.k8s.io/controller-reconciler/pkg/reconciler/manager/k8s/manager.go

```diff
@@ -116,6 +117,23 @@ func isReferringSameObject(a, b metav1.OwnerReference) bool {
 	return aGV == bGV && a.Kind == b.Kind && a.Name == b.Name
 }
 
+func isKindFormatUpdate(a, b metav1.OwnerReference) bool {
```

Comment:
Not sure if I understand what this function is trying to do. How do we know b is using old format for the kind?

Reply:
Rename this to needUpdateOwnerRefKindFormat(). 
Add comments to give context on old vs new format for GKE 1.19 vs 1.20+. Explain what returning true mean and what the call should do when get back a true.

Reply:
Acknowledged